workshops/models.py

```diff
@@ -108,7 +108,7 @@ class Person(AbstractBaseUser, PermissionsMixin):
     email       = models.CharField(max_length=STR_LONG, unique=True, null=True, blank=True)
     gender      = models.CharField(max_length=1, choices=GENDER_CHOICES, null=True, blank=True)
     may_contact = models.BooleanField(default=True)
-    airport     = models.ForeignKey(Airport, null=True, blank=True)
+    airport     = models.ForeignKey(Airport, null=True, blank=True, on_delete=models.SET_NULL)
```

Comment:
Hmm.  If a `Person` references an `Airport` I don't think we should be deleting the airport.  Can we have a check in the airport-delete view that says "You can't delete this airport, because it's being used by: {list-of-linkers}”?  Then someone interested in deleting the airport can use information about the to-be-deleted airport to figure out what to do about the existing linkers.  For duplicate airport entries (should be impossible with unique `iata` and `fullname` fields?) that may be “shift them all over to {alternative-iata}”;